dbms/src/Storages/MergeTree/MergeTreeDataPart.h

```diff
@@ -86,6 +86,7 @@ struct MergeTreeDataPart
 
     const MergeTreeData & storage;
 
+    String path;
```

Comment:
Bad name. We have bunch of different paths.

Reply:
Hm, 
path + relative_path + name = full_path
What name should we use for path?
Can you suggest?

Reply:
data_path + relative_path + name.
Mark it as private
add functions:
getFullPath() -> data_path + relative_path + name
getName() -> name
setRelativePath()